spec/rails_state_machine/state_machine/event_spec.rb

```diff
@@ -0,0 +1,103 @@
+describe RailsStateMachine::StateMachine do
+
+  shared_examples 'an invalid record in new state' do
+    describe '#<event_name>' do
+      let(:error_messages) { parcel.errors.messages }
+
+      it 'does not save the record' do
+        expect(parcel.state).to eq('empty')
+        expect(parcel.may_pack?).to be(true)
+        expect(parcel.pack).to eq(false)
+        expect(error_messages[:weight]).to include("can't be blank")
+        expect(parcel.state).to eq('empty')
+      end
+    end
+
+    describe '#<event_name>!' do
+      it 'raises an error' do
+        expect(parcel.state).to eq('empty')
+        expect(parcel.may_pack?).to be(true)
+        expect { parcel.pack! }.to raise_error(
+          ActiveRecord::RecordInvalid,
+          "Validation failed: Weight can't be blank"
```

Comment:
This fails for me because the transition is also invalid:

```
expected ActiveRecord::RecordInvalid with "Validation failed: Weight can't be blank", got #<ActiveRecord::RecordInvalid: Validation failed: Weight can't be blank, State event pack may not transition from filled>
```

Maybe just omit the explicit error message, or use a regular expression instead, like `/Validation failed: Weight can't be blank/`